specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/examples/AvailabilityStatuses_ListByResourceGroup.json

```diff
@@ -0,0 +1,67 @@
+﻿{
+	"parameters": {
+		"subscriptionId": "subscriptionId",
+		"resourceGroupName": "resourceGroupName",
+		"api-version": "2015-01-01",
+		"$expand": "recommendedactions"
+	},
+	"responses": {
+		"200": {
+			"body": {
+				"value": [{
+					"id": "<resourceId>/providers/Microsoft.ResourceHealth/AvailabilityStatueses/current",
+					"name": "current",
+					"type": "Microsoft.ResourceHealth/AvailabilityStatuses",
+					"location": "eastus",
+					"properties": {
+						"availabilityState": "Available",
+						"summary": "Vm is available",
+						"reasonType": "Unplanned",
+						"reasonChronicity": "Persistent",
+						"detailedStatus": "We have not seen any issues with your virtual machine",
+						"occuredTime": "2016-03-29T09:12:00Z",
+						"reportedTime": "2016-05-04T14:11:29.7598931Z",
+						"recentlyResolved": {
```

Comment:
recentlyResolvedState

Reply:
This is an existing contract and do not want to change it.

Reply:
But `recentlyResolved` is not defined in the spec. I only found `recentlyResolvedState`

Reply:
@HimanshuMisra1  Can you please take a look at this?